### ISSUE TITLE
Unable to train the model.

### ISSUE THREAD

**Issue by kumarUjjawal** [zaidalyafeai/Browser-Sentiment-Classification#2]
I'm getting this error when trying to train the model.![screen shot 2018-04-20 at 6 09 43 pm](https://user-images.githubusercontent.com/10569915/39051363-09c5a1a4-44c6-11e8-95a3-b9dc5158212b.png)

**Comment by zaidalyafeai** [comment]
Could you post the versions of keras, TensorFlow and python you have ?

**Comment by kumarUjjawal** [comment]
Tensorflow = 1.7.0
keras = 2.1.5
Python = 2.7

**Comment by zaidalyafeai** [comment]
Try with python = 3.5

**Comment by kashyap32** [comment]
same error with python 3.5

**Comment by neohack22** [comment]
I have the same indeed, from Jupyther notebook as well...

**Comment by sid321axn** [comment]
I am also facing the same problem

**Comment by harshel** [comment]
> 
> 
> I am also facing the same problem

The output labels that we have are contained in 'y_train' which is a list. Convert the 'y_train' list into a numpy array and feed that array while fitting the model.
